.doc_gen/templates/zonbook/example_language_template.xml

```diff
@@ -53,7 +53,7 @@
                 {{- end}}
                 {{- if .SnippetTags}}
                 <!-- The following line break must be preserved and left-justified exactly as-is, to keep snippets looking good. -->
-                <programlisting language="{{.Syntax}}">{{range .SnippetTags}}<xi:include parse="text" href="{{$include_base}}snippets/{{.}}.txt"/>
+                <programlisting language="{{.Syntax}}">{{range .SnippetTags}}<xi:include parse="{{- if hasSuffix . ".xml"}}xml{{- else}}text{{- end}}" href="{{$include_base}}snippets/{{.}}.txt"/>
```

Comment:
This'll probably work functionally, but it strikes me as strange that we are using .txt as the suffix for .xml files, so I'd leave of the .txt in that case (although I guess that means writing files without the .txt as well, so maybe not worth the effort?).

Reply:
No, .txt as a suffix is deeply embedded in the logic of every piece of the tool chain. We're not up for that much refactoring at this time.